src/interfaces/ICuratedModule.sol

```diff
@@ -0,0 +1,22 @@
+// SPDX-FileCopyrightText: 2025 Lido <info@lido.fi>
+// SPDX-License-Identifier: GPL-3.0
+
+pragma solidity 0.8.24;
+
+import { ICSModule } from "./ICSModule.sol";
+
+interface ICuratedModule is ICSModule {
+    error NotImplemented();
+
+    function NODE_OWNER_ADMIN_ROLE() external view returns (bytes32);
+
+    /// @notice Change both reward and manager addresses of a node operator.
+    /// @param nodeOperatorId ID of the Node Operator
+    /// @param newRewardAddress New reward address
+    /// @param newManagerAddress New manager address
+    function changeNodeOperatorAddresses(
+        uint256 nodeOperatorId,
+        address newManagerAddress,
+        address newRewardAddress
```

Comment:
```suggestion
    /// @param managerAddress New manager address
    /// @param rewardAddress New reward address
    function changeNodeOperatorAddresses(
        uint256 nodeOperatorId,
        address managerAddress,
        address rewardAddress
```

Reply:
The same in `src/lib/NOAddresses.sol`

Reply:
I disagree. It less ambiguous and works better at the rest of function code.

Reply:
I mean you have different arg names + blended order in natspec